src/components/autofill/autofill.ts

```diff
@@ -100,15 +99,21 @@ export class Autofill extends Component<Props, SpreadsheetChildEnv> {
       this.env.model.dispatch("AUTOFILL");
     };
 
-    const onMouseMove = (ev: MouseEvent) => {
-      const position = gridOverlayPosition();
+    const onMouseMove = (col: HeaderIndex, row: HeaderIndex, ev: MouseEvent) => {
       const { offsetY, offsetX } = this.env.model.getters.getActiveSheetScrollInfo();
-      this.state.position = {
-        left: ev.clientX - start.left + offsetX,
-        top: ev.clientY - start.top + offsetY,
-      };
-      const col = this.env.model.getters.getColIndex(ev.clientX - position.left);
-      const row = this.env.model.getters.getRowIndex(ev.clientY - position.top);
+      if (this.props.position.left > 0 && this.props.position.top > 0) {
+        // autofill selected region still visible
+        this.state.position = {
+          left: ev.clientX - start.left + offsetX,
+          top: ev.clientY - start.top + offsetY,
+        };
+      } else {
+        this.state.position = {
+          left: ev.clientX,
+          // TODO: 60 is hardcoded. We need more advanced method to determine the value, considering the autofill CSS position will become negative numbers when the autofill selected region is invisible, and it has impact on tooltip & handler whose CSS position is absolute which relates to the position of autofill itself.
```

Comment:
@LucasLefevre Agreed, and I tried several ways but using `ev.clientX` and `ev.clientY` here is the simplest and most accurate solution IMHO. 
Handler and tooltip are wrapped inside the autofill div and the position of autofill div is decided by the visible rect. Once the rect is invisible both `left` and `top` become `-4`. 
We can surely change the position here but change to what? The mouse location? It seems the same as we use `clientX` and `clientY` here. And we don't care where the autofill div is (because it will be hidden visually).